services-camel/src/test/java/edu/unc/lib/dl/services/camel/cdrEvents/CdrEventRouterTest.java

```diff
@@ -99,15 +99,15 @@ public void testPublishFilter() throws Exception {
     public void testReindexFilter() throws Exception {
```

Comment:
Reindex events aren't being sent to the cdr events queue anymore after OperationsMessageSender was split, right?  So I think this test can go